platform/mellanox/docker-syncd-mlnx.mk

```diff
@@ -3,7 +3,9 @@
 DOCKER_SYNCD_PLATFORM_CODE = mlnx
 include $(PLATFORM_PATH)/../template/docker-syncd-base.mk
 
-$(DOCKER_SYNCD_BASE)_DEPENDS += $(SYNCD) $(PYTHON_SDK_API) $(MFT)
+$(DOCKER_SYNCD_BASE)_DEPENDS += $(SYNCD) $(PYTHON_SDK_API)
+
+$(DOCKER_SYNCD_BASE)_FILES += $(ISSU_VERSION_FILE)
```

Comment:
This sets issu to build with syncd so that it builds in the buster environment along with the syncd dependencies. 

MFT is also removed as a dependency to syncd as syncd currently only builds in buster but MFT requires linux kernel heads to be built which are not capable of being built in the buster environment. As far as I can tell this has no impact on switch functionality.

Reply:
MFT needs for techsupport and shouldn't be removed.

Reply:
Resolved.